docs/docs/examples-tutorials/recipes/running-tracetest-with-honeycomb.md

```diff
@@ -0,0 +1,340 @@
+        target: /app/tracetest.yaml
+      - type: bind
+        source: ./tracetest/tracetest-provision.yaml
+        target: /app/provision.yaml
+    command: --provisioning-file /app/provision.yaml
+    healthcheck:
+      test: ["CMD", "wget", "--spider", "localhost:11633"]
+      interval: 1s
+      timeout: 3s
+      retries: 60
+    depends_on:
+      postgres:
+        condition: service_healthy
+      otel-collector:
+        condition: service_started
+    environment:
+      TRACETEST_DEV: ${TRACETEST_DEV}
+
+  postgres:
+    image: postgres:14
+    environment:
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+    healthcheck:
+      test: pg_isready -U "$$POSTGRES_USER" -d "$$POSTGRES_DB"
+      interval: 1s
+      timeout: 5s
+      retries: 60
+
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.68.0
+    restart: unless-stopped
+    command:
+      - "--config"
+      - "/otel-local-config.yaml"
+    volumes:
+      - ./tracetest/collector.config.yaml:/otel-local-config.yaml
+
+```
+
+Tracetest depends on both Postgres and the OpenTelemetry Collector. Both Tracetest and the OpenTelemetry Collector require config files to be loaded via a volume. The volumes are mapped from the root directory into the `tracetest` directory and the respective config files.
+
+To start both the Node.js App and Tracetest, run this command:
+
+```bash
+docker-compose up # add --build if the images are not built already
+```
+
+The `tracetest-config.yaml` file contains the basic setup of connecting Tracetest to the Postgres instance.
+
+```yaml
+# tracetest-config.yaml
+
+---
+postgres:
+  host: postgres
+  user: postgres
+  password: postgres
+  port: 5432
+  dbname: postgres
+  params: sslmode=disable
+
+```
+
+The `tracetest-provision.yaml` file contains the data store setup. The data store is set to `Honeycomb` meaning the traces will be received by Tracetest OTLP API and stored in Tracetest itself.
+
+```yaml
+# tracetest-provision.yaml
+---
+type: DataStore
+spec:
+  name: Honeycomb
+  type: honeycomb
+```
+
+**How to Send Traces to Tracetest and Honeycomb**
+
+The `collector.config.yaml` explains that. It receives traces via either `grpc` or `http`. Then, exports them to Tracetest's OTLP endpoint `tracetest:4317` in one pipeline, and to Honeycomb in another.
+
+Make sure to add your Honeycomb access token in the headers of the `otlp/ls` exporter.
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  batch:
+    timeout: 100ms
+
+exporters:
+  logging:
+    logLevel: debug
+  # OTLP for Tracetest
+  otlp/tt:
+    endpoint: tracetest:4317 # Send traces to Tracetest. Read more in docs here:  https://docs.tracetest.io/configuration/connecting-to-data-stores/opentelemetry-collector
+    tls:
+      insecure: true
+  # OTLP for Honeycomb
+  otlp/hc:
+    endpoint: "api.honeycomb.io:443"
+    headers:
+      "x-honeycomb-team": <HONEYCOMB_API_KEY>
```

Comment:
Should we add the api key in the notion page @danielbdias created? https://www.notion.so/kubeshop/Vendor-keys-db37b8db460345b6be3ca4d3c4345940

Reply:
Yep